.rubocop.yml

```diff
@@ -49,6 +49,12 @@ RSpec/DescribeClass:
 RSpec/ExampleLength:
   Enabled: false
 
+# FIXME: Find a better way to block output during specs, and fix relevant examples.
+RSpec/ExpectOutput:
```

Comment:
Thanks, @backus. I'm going with simple suppressions for now and will revisit this later and see if I can make your suggestion work for all cases.